eng/Build.ps1

```diff
@@ -543,6 +559,29 @@ try {
         TestUsingNUnit -testProject "$RepoRoot\vsintegration\tests\UnitTests\VisualFSharp.UnitTests.fsproj" -targetFramework $desktopTargetFramework
     }
 
+    # verify nupkgs have access to the source code
+    $nupkgtestFailed = $false
+    if ($testpack) {
+        # Fetch soucelink test
+        $dotnetPath = InitializeDotNetCli
+        $dotnetExe = Join-Path $dotnetPath "dotnet.exe"
+        try {
+            Exec-Console $dotnetExe "tool install sourcelink --tool-path ""$dotnetPath"""
```

Comment:
could/would we  be able to use normal dotnet tool-manifests here instead? that would also allow contributors to more easily  do `dotnet sourcelink test <dll>` on any dll they want in the solution.

Reply:
@baronfel this is just a local tool install for the repo build.  developers should be able to do a dotnet install sourcelink on their machines whenever they like, and it will be unimpacted by this.

Reply:
@baronfel this does however point up a problem, given that, and I didn't know this until minutes ago, and I am definitely crying over this ... can return "the path to the global dotnet.exe" if the requested sdk is already installed.

Reply:
I will submit a fix PR.

Reply:
The fact that @brettfo suspected and proved this behavior, demonstrates how cynics are a valuable resource and not to be trifled with.

Reply:
Brain exploded:
![image](https://user-images.githubusercontent.com/5175830/90440085-81254200-e08b-11ea-982e-a7b286e87a77.png)